src/main/resources/graphql/schema.graphqls

```diff
@@ -0,0 +1,22 @@
+type Query {
+    #user
+    users:[User]
+    user(id:ID!):User
+    #team
+    teams(count: Int):[Team]
+    team(id:ID!):Team
+    #tournament
+    tournaments:[Tournament]
+    tournament(id:ID!):Tournament
+}
+
+type Mutation {
```

Comment:
position of the comments is changed when showing the schema, so it currently looks weird

Reply:
Agreed, looking for a way to display it nicely